.github/workflows/workflows/codeql.yml

```diff
@@ -0,0 +1,46 @@
+name: "Code Scanning - Action"
```

Comment:
@jhutchings1 just checking this path is correct? .github/**workflows/workflows**/codeql.yml?

Reply:
Good catch @brendankowitz! That's an issue in my script I'll have to look into. I'll patch it.

Reply:
Fixed!